Project-Hotpot/Project-Hotpot/HomeViewController.swift

```diff
@@ -98,15 +74,52 @@ class HomeViewController: UIViewController {
         }
     }
     
+    // MARK: - helper functions
+    
     func resetCard(){
         NSLog("resetting")
         UIView.animate(withDuration: 0.2, animations: {
             self.card.center = self.view.center
-//            self.ca
+            self.songTitleLabel.text = self.api_instance.curr_song_label
             
         })
     }
     
+    func playSong(){
+        //resume the audio
+        api_instance.appRemote.playerAPI?.resume()
+        //change the button image
+        let newIcon = UIImage(systemName: "pause.circle.fill")
+        self.playButton.setImage(newIcon, for:.normal)
+    }
+    
+    func pauseSong(){
+        //pause the audio
+        api_instance.appRemote.playerAPI?.pause()
+        //change the button image
+        let newIcon = UIImage(systemName: "play.circle.fill")
+        self.playButton.setImage(newIcon, for:.normal)
+        
+    }
+    
+    func resetSong(){
```

Comment:
It's best to have a space between your parentheses and opening bracket (`() {`).

Unfortunately, Xcode doesn't have great auto-formatting like Android Studio does (Ctrl+i really only fixes indentation and a handful of styling issues). If you're curious, you could install SwiftLint (https://swiftpackageindex.com/realm/SwiftLint, they have a CocoaPods version) to prettify your code.